src/c2goto/library/builtin_libs.c

```diff
@@ -16,4 +16,36 @@ int __ESBMC_sync_fetch_and_add(int *ptr, int value)
   *ptr += value;
   __ESBMC_atomic_end();
   return initial;
-}
\ No newline at end of file
+}
+
+int __ESBMC_parity(unsigned int value)
```

Comment:
Why do we need to use `__ESBMC_` here? Isn't parity a built-in function?

Reply:
Yes, it is a builtin. Actually, I was just following the steps of some other builtins, like sync_fetch_and_add.
In wich cases do we do this?